spec/requests/self_assessment_pages_spec.rb

```diff
@@ -17,7 +17,8 @@
       end
 
       it "lets AC create a self assessment" do
-        fill_in 'Response', with: 'This is a self-assessment.'
+        fill_in 'Performance Assessment', with: 'This is a self-assessment.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.